src/main/java/com/epam/wl/entities/UserOrder.java

```diff
@@ -4,13 +4,13 @@
 import lombok.Data;
 
 @Data
-public class Order {
+public class UserOrder {
     private int id;
     private int editionId;
```

Comment:
field "editionID" is called "bookID" now

Reply:
Done